packages/reports/addon/templates/components/filter-builders/dimension.hbs

```diff
@@ -0,0 +1,40 @@
+<span class='filter-builder-dimension__subject'>
+    {{displayName}}
+</span>
+{{#power-select
+    class='filter-builder-dimension__operator'
+    dropdownClass='filter-builder-dimension__operator-dropdown'
+    options=supportedOperators
+    selected=filter.operator
+    searchEnabled=false
+    tagName='span'
+    triggerClass='filter-builder-dimension__select-trigger'
+    onchange=(action 'setOperator')
+    as | operator |
+}}
+    {{operator.longName}}
+{{/power-select}}
+{{#if showFields}}
+  {{#power-select
+    searchEnabled=false
+    options=fields
+    selected=filter.field
+    tagName='span'
+    class='filter-builder-dimension__field'
+    dropdownClass='filter-builder-dimension__field-dropdown'
+    triggerClass='filter-builder-dimension__select-trigger'
+    matchTriggerWidth=false
+    onchange=(action 'setField')
+    as | field |
+  }}
+    field: {{field}}
```

Comment:
As discussed in the sync up, can this be just `{{field}}`

Reply:
Yeah I did that because it bugs me there is no label for the form control. I see how it's weird though, I will respect the council's decision.